cmake/compile_definitions/common.cmake

```diff
@@ -108,6 +106,8 @@ set(SUNSHINE_TARGET_FILES
         "${CMAKE_SOURCE_DIR}/src/round_robin.h"
         "${CMAKE_SOURCE_DIR}/src/stat_trackers.h"
         "${CMAKE_SOURCE_DIR}/src/stat_trackers.cpp"
+        "${CMAKE_SOURCE_DIR}/src/rswrapper.h"
+        "${CMAKE_SOURCE_DIR}/src/rswrapper.c"
```

Comment:
nit: Could these be placed alphabetically?

Reply:
Should I reorder them all? The list order currently looks arbitrary.

Reply:
Oh, I didn't expand the code when I made this comment. It's fine as is. I think we'll do a glob pattern in the future to make it easier.